content/md/en/docs/maintain/monitor.md

```diff
@@ -4,59 +4,174 @@ description:
 keywords:
 ---
 
-Monitoring network operations and performance is an important part of maintaining a healthy, stable, and secure parachain ecosystem.
+Monitoring network operations and performance is an important part of maintaining a healthy, stable, and secure ecosystem.
 By default, Substrate nodes provide low-level **telemetry** components to automatically collect and transmit detailed information about the operation of each node in the network.
 Telemetry is the underlying mechanism—sensors, probes, or instrumentation—for collecting and transmitting data points as they occur to an endpoint so that you can observe, monitor, and alert on system behavior.
```

Comment:
I'd probably say something closer to "This information is referred to as telemetry" (since you summarised it well above). I don't feel like this sentense really adds anything :)

(PS I love this approach of having each sentence on a newline..)

Reply:
modified to focus on the backend > endpoint